lms/djangoapps/teams/templates/teams/teams.html

```diff
@@ -43,6 +43,9 @@
     TeamsTabFactory({
         courseID: '${six.text_type(course.id) | n, js_escaped_string}',
         topics: ${topics | n, dump_js_escaped_json},
+        hasManagedTopic: ${has_managed_teamset | n, dump_js_escaped_json},
+        hasPublicManagedTopic: ${has_public_managed_teamset | n, dump_js_escaped_json},
+        hasOpenTopic: ${has_open_teamset | n, dump_js_escaped_json},
```

Comment:
Naming suggestion: make plural so it doesn't seem like we can have only one of each type of topic
```suggestion
        hasManagedTopics: ${has_managed_teamset | n, dump_js_escaped_json},
        hasPublicManagedTopics: ${has_public_managed_teamset | n, dump_js_escaped_json},
        hasOpenTopics: ${has_open_teamset | n, dump_js_escaped_json},
```

Reply:
I thought to make them singular because we're  only worried about there being at least one, I didn't want these to somehow imply that there were multiple of a type.  

and I thought hasAtLeastOne*Topic was too long of a name xD